### PR TITLE
Feature/#85 Errorhandling

### PR DESCRIPTION
**Pull request by flxtagi**
lets test with this configuration. not tested all Errors.

### CODE REVIEW

**Comment by flxtagi** [comment]
catch really 500 errors ? 
static getDerivedStateFromError(error: unknown) {

500 Qwacker API request failed with status 500 (Internal Server Error).

File: src/pages/index.tsx
Comment (rudigier) [inline src/pages/index.tsx:1]: Guess throw a error would be better that show this Custom500

**Review by rudigier — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+import Custom500Page from './500';
 
 export default function PageHome({
 	postCount: postCount,
 	posts,
 	error,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
 	if (error) {
-		return <ErrorPage statusCode={500} title={error} />;
+		return <Custom500Page errorInfo={error} />;
```
Comment (flxtagi) [inline src/pages/index.tsx:16]: lets test that if error page needs here

**Review by rudigier — APPROVED** [review]
Thanks for the changes. Looks good. Let's see how vercel likes it.

**Pull request merged**